lib/cli/ui/stdout_router.rb

```diff
@@ -15,13 +15,15 @@ def initialize(stream, name)
         end
 
         def write(*args)
-          if auto_frame_inset?
-            str = args[0].dup # unfreeze
-            str = str.force_encoding(Encoding::UTF_8)
-            str = apply_line_prefix(str, CLI::UI::Frame.prefix)
-            args[0] = str
-          else
-            @pending_newline = false
+          args = args.map do |str|
+            if auto_frame_inset?
+              str = str.dup # unfreeze
+              str = str.force_encoding(Encoding::UTF_8)
+              apply_line_prefix(str, CLI::UI::Frame.prefix)
+            else
+              @pending_newline = false
```

Comment:
Should this be only for the last arg? or?

Reply:
No, each.